SwiftStyleGuide.swift

```diff
@@ -164,12 +159,11 @@ private extension String {
 
 }
 
-// swiftlint:disable identifier_name
 // Use one line per let in a guard with multiple lets.
-let maybe🌮 = restaurant.order("🥗")
-let maybe🥤 = restaurant.order("☕️")
-guard let veggieTaco = maybe🌮,
-      let coffee = maybe🥤 else {
+let taco = restaurant.order("taco")
```

Comment:
WHERE DID THE EMOJIS GO??

Reply:
LOL I didn't wanna deal with swiftlint complaining about variable length, but I can bring them back

Reply:
It's all good 😄